### ISSUE TITLE
Test failure on 1030-invalid-wkb-polygons.py

### ISSUE THREAD

**Issue by pnorman** [tilezen/vector-datasource#1438]
I got this in https://github.com/tilezen/vector-datasource/pull/1435#issuecomment-346169071, but also see it locally

```
======================================================================
ERROR: test_mvt_feature_validity (integration-test.1030-invalid-wkb-polygons.InvalidWkbPolygons)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pnorman/osm/tilezen/vector-datasource/integration-test/1030-invalid-wkb-polygons.py", line 62, in test_mvt_feature_validity
    with self.features_in_mvt_layer(5, 17, 9, 'water') as features:
  File "/usr/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "integration-test/__init__.py", line 1065, in inner
    format_mvt(io, pfl, z, bounds_merc, bounds_lnglat, extent)
TypeError: format_mvt() takes exactly 5 arguments (6 given)

----------------------------------------------------------------------
```

**Comment by zerebubuth** [comment]
This should have been fixed in #1436. It's passing for me locally and [latest master on CircleCI](https://circleci.com/gh/tilezen/vector-datasource/2118). With so many repos, it's possible that our versions aren't completely in sync. If you've merged or rebased on top of #1436 and are still seeing this, please let me know the exact commit IDs so that I can try to replicate.

**Comment by pnorman** [comment]
Yep, fixed now.
